src/main/java/duke/commands/AddDeadlineCommand.java

```diff
@@ -0,0 +1,65 @@
+package duke.commands;
+
+import java.io.IOException;
+
+import duke.main.DukeException;
+import duke.main.Parser;
+import duke.main.Storage;
+import duke.main.TaskList;
+import duke.main.Ui;
+import duke.tasks.Deadline;
+
+
+/**
+ * AddDeadlineCommand is a Command.
+ * This Command is used to add Deadlines to the TaskList.
+ */
+public class AddDeadlineCommand extends Command<String> {
```

Comment:
Perhaps it might be better to rename AddDeadlineCommand to just DeadlineCommand?

Reply:
Using verbs as class names might get confusing when calling constructors and methods